LICENSE.md

```diff
@@ -0,0 +1 @@
+# TODO
```

Comment:
No, I mean the plain-text files [already exist](https://github.com/Gene42/phenotips-niaid/blob/master/LICENSE) [in the project](https://github.com/Gene42/phenotips-niaid/blob/master/NOTICE).